### ISSUE TITLE
"warning" RST blocks not not showing up correctly in the documentation

### ISSUE THREAD

**Issue by dwf** [mila-udem/blocks#918]
[One example](https://blocks.readthedocs.org/en/latest/api/bricks.html?highlight=warning#blocks.bricks.parallel.Merge), there may be others.

**Comment by bartvm** [comment]
Right, the warning blocks (and any other blocks or text really) should be
part of the main text/description i.e. before the parameters, attributes,
etc. sections, otherwise weird things happen.

**Comment by rizar** [comment]
Permuting a few sections in the documentation seems to be a CCW level task.
